src/components/PostDetails.tsx

```diff
@@ -132,7 +62,9 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
                     type="button"
                     className="delete is-small"
                     aria-label="delete"
-                    onClick={() => deleteComment(comment.id)}
+                    onClick={() =>
+                      dispatch(commentsActions.removeComment(comment.id))
+                    }
```

Comment:
There is a syntax error here. The closing curly brace for the onClick function should be placed before the closing parenthesis of the JSX element.